api/src/pcapi/sandboxes/scripts/creators/industrial/create_industrial_commercial_gestures.py

```diff
@@ -0,0 +1,433 @@
+            booking=booking,
+            author=author_user,
+            origin="create_industrial_commercial_gestures in industrial sandbox",
+        )
+        finance_api.validate_finance_commercial_gesture(
+            finance_incident=commercial_gesture,
+            force_debit_note=False,
+            author=author_user,
+        )
+
+        # TODO(amine) figure out why this is mandatory to be able to fetch the related pricings in `_price_event` function
+        # tried session.expire_all() and session.commit() but no effect
+        [p.id for p in booking.pricings]  # # pylint: disable=pointless-statement
+
+        # Pricing of the commercial gestures
+        for booking_finance_incident in commercial_gesture.booking_finance_incidents:
+            for event in booking_finance_incident.finance_events:
+                finance_api.price_event(event)
+
+
+def _create_partial_commercial_gesture_multiple_individual_offer(
+    author_user: users_models.User,
+    venue: offerers_models.Venue,
+    size: int,
+) -> None:
+    # Create the users
+    users = users_factories.BeneficiaryGrant18Factory.create_batch(size)
+    # Empty the balances
+    for user in users:
+        booking = bookings_factories.BookingFactory(
+            user=user,
+            quantity=40,
+            stock__price=decimal.Decimal("5.0"),
+            stock__offer__venue=venue,
+        )  # 200€
+        bookings_api.mark_as_used(
+            booking=booking,
+            validation_author_type=bookings_models.BookingValidationAuthorType.OFFERER,
+        )
+
+        for finance_event in booking.finance_events:
+            finance_api.price_event(finance_event)
+
+    # Create the bookings to cancel
+    bookings = []
+
+    for i, user in enumerate(users):
+        booking = bookings_factories.BookingFactory(
+            user=user,
+            stock__offer__venue=venue,
+            stock__price=decimal.Decimal("10.1") + decimal.Decimal(i),
+            quantity=1,
+        )
+        bookings_api.mark_as_used(
+            booking=booking,
+            validation_author_type=bookings_models.BookingValidationAuthorType.OFFERER,
+        )
+        for finance_event in booking.finance_events:
+            finance_api.price_event(finance_event)
+
+        bookings_api.mark_as_cancelled(
+            booking=booking,
+            reason=bookings_models.BookingCancellationReasons.BACKOFFICE,
+        )
+
+        for finance_event in booking.finance_events:
+            if finance_event.status == finance_models.FinanceEventStatus.CANCELLED:
+                finance_api.price_event(finance_event)
+
+        bookings.append(booking)
+
+    # Empty the remaining credits
+    for user in users:
+        booking = bookings_factories.BookingFactory(
+            user=user,
+            quantity=1,
+            stock__price=decimal.Decimal("99.9"),  # keep the user with only 10cents
```

Comment:
Tu peux garder le même commentaire je pense, `# Empty the remaining credits leaving only 10 cents in each balance`, histoire d'être uniforme